### PR TITLE
Update run 3 datasets.

### PR DESCRIPTION
**Pull request by riga**
This PR updates the list of run 3 datasets in our config and adds a few additional minor improvements.

- datasets defined in the config are now complete and synchronized (although slight changes might follow in the future)
- there is a new config for the 22 post campaign
- other configs have been commented out and should be un-commented as soon as all required datasets exist for them (@nprouvost once 22 pre is ready (again), this can be changed)
- there is a sync-specific config now that points to the datasets and files we agreed upon with other groups (@Bogdan-Wiederspan)
- cmsdb points to our feature branch (https://github.com/uhh-cms/cmsdb/tree/campaign/run3_2022_postEE_nano_uhh_v12) right now but should be set to master once https://github.com/uhh-cms/cmsdb/pull/66 and https://github.com/uhh-cms/cmsdb/pull/67 are merged. However, we should already merge _this_ one to start.

### CODE REVIEW

File: hbt/config/configs_hbt.py [hunk 1]
```diff
@@ -43,6 +44,10 @@ def add_config(
     assert run in {2, 3}
     assert year in {2016, 2017, 2018, 2022, 2023}
 
+    # renamings
+    sync_requested = sync
+    del sync
+
```
Comment (Bogdan-Wiederspan) [inline hbt/config/configs_hbt.py:50]: I don't like that we rename the variable right after declaring it. I find that `sync` as flag is clear enough. 
```suggestion
```

Comment (Bogdan-Wiederspan) [reply hbt/config/configs_hbt.py:50]: Everything else looks good to me. Its nice to see that we now have a lazy_loader for configs.

Comment (riga) [reply hbt/config/configs_hbt.py:50]: "now" = July 15 ;)

Comment (riga) [reply hbt/config/configs_hbt.py]: Also, agreed. I renamed it to `sync_mode` as `sync` itself can be read as a verb / instruction.

**Review by Bogdan-Wiederspan — APPROVED** [review]

**Pull request merged**